### PR TITLE
Add gradle

### PR DESCRIPTION
**Pull request by Muirrum**
Gradle library stuff

### CODE REVIEW

**Comment by Muirrum** [comment]
Reasoning: WPILib and the code uses Gradle, and GitHub has a built-in package registry that we could use quite simply.

File: .gitignore [hunk 1]
```diff
@@ -1,168 +1,5 @@
-# Created by https://www.gitignore.io/api/c++,java,linux,macos,gradle,windows,visualstudiocode
```
Comment (Muirrum) [inline .gitignore:1]: We can definitely shrink this down, this was generated by gradle

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * User Manual available at https://docs.gradle.org/5.5.1-20190724234647+0000/userguide/java_library_plugin.html
+ */
+
+plugins {
+    // Apply the java-library plugin to add support for Java Library
+    id 'java-library'
+}
+// Set library info
+version = "0.0.1"
+group = "com.frc4905"
```
Comment (Muirrum) [inline build.gradle:16]: I kinda just picked a [versioning system](https://semver.org) and group (based on the website)

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+dependencies {
+    // This dependency is exported to consumers, that is to say found on their compile classpath.
+    api 'org.apache.commons:commons-math3:3.6.1'
+
+    // This dependency is used internally, and not exposed to consumers on their own compile classpath.
+    implementation 'com.google.guava:guava:27.1-jre'
+
+    // Use JUnit test framework
+    testImplementation 'junit:junit:4.12'
+}
```
Comment (Muirrum) [inline build.gradle:51]: I don't think we need any of these except junit, can be removed at other people's discretion

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-bin.zip
```
Comment (Muirrum) [inline gradle/wrapper/gradle-wrapper.properties:3]: This is the latest gradle version, wrapper keeps us on this one

File: src/main/java/AlienLib/Library.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+/*
```
Comment (Muirrum) [inline src/main/java/AlienLib/Library.java:1]: Both this and the equivalent test files can be removed. These were autogenerated by gradle.

**Review by themccubbins — APPROVED** [review]

**Pull request merged**